// arg_named_too_many.orng:6:11: error: function takes 1 parameter, 2 arguments given
//     f(.x = 4, .y = 4)
//          ^
// 
fn main() -> Int {
    f(.x = 4, .y = 4)
}

fn f(x: Int) -> Int {
    x
}